packages/seeder/src/seeder-demo.js

```diff
@@ -2,11 +2,16 @@ const Seeder = require('./seeder')
 
 const main = async () => {
   const s = new Seeder()
-  await s.seed(['aa27834b41fa78c091ddb066e889202775bb3b5f57586c69d709be9f7231fccf'])
+  await s.seed(['2a09008a55c8f4bcc14987ed43331675611a2b506859ebfdd050b0f41746ad02'])
   s.on('drive-progress', (...args) => console.log('progress', args))
-  s.on('drive-finish', (dkey, ...args) => {
+  s.on('drive-finish', async (dkey, ...args) => {
     console.log('finish download', dkey)
     console.log(`Content available in: ${s.opts.storageLocation}`)
+    const stats = await s.allStats()
```

Comment:
Why not call to `stat(dkey)` here?

Reply:
Yeah, that would be better. I was just toying around with `allStats()` method.